
# HG changeset patch
# User Brendan Cully <brendan@kublai.com>
# Date 1266957745 28800
# Node ID 57124ea5592ee350af277b543bfedf04d3cd0ace
# Parent  f17302c1022979c654170a0ca0f5a4cc8ffd9eb5
Don't use freed ctx.path when updating utime of target mailbox in save-message.
Thanks to Sean Boudreau for the bug report.

diff -r f17302c10229 -r 57124ea5592e commands.c
--- a/commands.c	Mon Feb 22 23:01:39 2010 -0800
+++ b/commands.c	Tue Feb 23 12:42:25 2010 -0800
@@ -863,7 +863,7 @@
     mx_close_mailbox (&ctx, NULL);
 
     if (need_buffy_cleanup)
-      mutt_buffy_cleanup (ctx.path, &st);
+      mutt_buffy_cleanup (buf, &st);
 
     mutt_clear_error ();
     return (0);

